AstSemantics.md

```diff
@@ -146,6 +146,21 @@ nested. This guarantees that all resulting control flow graphs are well-structur
     feature would allow efficient compilation of arbitrary irreducible control
     flow.
 
+## Floating Point Subnormal Handling
+
+  * `subnormal_mode`: Like [`block`](AstSemantics.md#control-flow-structures),
+     but with a mode attribute specifying special semantics for the handling of
+     subnormal values in all lexically contained [floating point operations][]
+     except `neg`, `abs`, and `copysign`, and [conversions][], except those in
+     contained `subnormal_mode` blocks. The mode may be one of:
+    * `standard`: standard semantics are followed
+    * `dont_care`: Subnormal values may or may not be interpreted as zero of the
+       same sign. Subnormal result values may or may not be replaced by any
+       subnormal value of the same sign, or zero of the same sign.
```

Comment:
What's "standard"? Wouldn't "IEEE754" be a better name?
I thought that we'd settled on "fastest" instead of "dont_care"?